app/src/main/res/layout/row_logs_filters_checkbox.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Removed altogether, as I've used programmatically defined [`Chip`s](https://developer.android.com/reference/com/google/android/material/chip/Chip) instead of checkboxes.